distrobuilder/main.go

```diff
@@ -136,6 +137,7 @@ func main() {
 		"Timeout in seconds"+"``")
 	app.PersistentFlags().BoolVar(&globalCmd.flagVersion, "version", false, "Print version number")
 	app.PersistentFlags().BoolVar(&globalCmd.flagDebug, "debug", false, "Enable debug output")
+	app.PersistentFlags().BoolVar(&globalCmd.flagOverlay, "overlay", true, "Enable overlay")
```

Comment:
Description string could be a bit more descriptive?
`Use a filesystem overlay for faster operations`?

Reply:
Hmm, actually, this is a bit confusing, maybe we should have it as `--disable-overlay`?

Reply:
A boolean defaulting to true is rather confusing ;)

Reply:
So I'd say, call it `--disable-overlay` and describe it as `Disable the use of filesystem overlays`?